include/cuco/detail/hash_functions/xxhash.cuh

```diff
@@ -60,22 +60,22 @@ namespace cuco::detail {
 template <typename Key>
 struct XXHash_32 {
  private:
-  static constexpr uint32_t prime1 = 0x9E3779B1U;
-  static constexpr uint32_t prime2 = 0x85EBCA77U;
-  static constexpr uint32_t prime3 = 0xC2B2AE3DU;
-  static constexpr uint32_t prime4 = 0x27D4EB2FU;
-  static constexpr uint32_t prime5 = 0x165667B1U;
+  static constexpr std::uint32_t prime1 = 0x9E3779B1u;
+  static constexpr std::uint32_t prime2 = 0x85EBCA77u;
+  static constexpr std::uint32_t prime3 = 0xC2B2AE3Du;
+  static constexpr std::uint32_t prime4 = 0x27D4EB2Fu;
+  static constexpr std::uint32_t prime5 = 0x165667B1u;
```

Comment:
I think @bdice's idea was to use lowercase consistently, e.g. 
```suggestion
  static constexpr std::uint32_t prime1 = 0x9e3779b1u;
  static constexpr std::uint32_t prime2 = 0x85ebca77u;
  static constexpr std::uint32_t prime3 = 0xc2b2ae3du;
  static constexpr std::uint32_t prime4 = 0x27d4eb2fu;
  static constexpr std::uint32_t prime5 = 0x165667b1u;
```
Not a hard requirement though

Reply:
fixed in [00e5b54](https://github.com/NVIDIA/cuCollections/pull/313/commits/00e5b54eb31aba90d396552c2552c3492c98fe0f)